### DISCUSSION TITLE
Maybe an extension that can redirect?

### DISCUSSION THREAD

**Discussion by KylomaskGamer** [TurboWarp/extensions#79] [Ideas]
That can be an idea for websites.

**Comment by softedco** [comment]
Isn't it already there as a cloud variable behaviour?

**Comment by WojtekCodesToday** [comment]
it's possible with js, but i'm unsure for GarboMuffin

**Comment by GarboMuffin** [comment]
This exists: https://extensions.turbowarp.org/true-fantom/network.js

**Reply by WojtekCodesToday** [reply]
Close the topic.

**Reply by KylomaskGamer** [reply]
> This exists: https://extensions.turbowarp.org/true-fantom/network.js

YAY now i can build website

**Comment by BlueDome77** [comment]
Close the topic please

**Comment by KylomaskGamer** [comment]
okey! :D

**Comment by ampelectrecuted** [comment]
@GarboMuffin this is in Network, can this be unpinned
(seems like it's pinned because it's the oldest discussion in this repo, but really just dust laying around)
